routes/api.php

```diff
@@ -13,7 +13,11 @@
 | is assigned the "api" middleware group. Enjoy building your API!
 |
 */
-
-Route::middleware('auth:api')->get('/user', function (Request $request) {
-    return $request->user();
-});
+Route::group(['middleware'=>['auth:api']], function()
+{
+Route::post('/newadmin',"AdminsController@NewAdmin");
+Route::post('/showadmin',"AdminsController@ShowAdmin");
```

Comment:
Read about the RESTful APIs and try to understand what is done wrong here.

_note: check route names and request methods_

Reply:
@eduardtovmasyan check this: https://habr.com/ru/post/351890/